packages/office-ui-fabric-react/src/components/Facepile/Facepile.Props.ts

```diff
@@ -76,6 +76,11 @@ export interface IFacepilePersona extends React.HTMLProps<HTMLButtonElement | HT
   onMouseOut?: (ev?: React.MouseEvent<HTMLElement>, persona?: IFacepilePersona) => void;
 
   /**
+   * If provided, the handler will be called when the component is mounted or unmounted
+   */
+  refHandler?: (el?: HTMLElement, persona?: IFacepilePersona) => void;
+
```

Comment:
I tried adding ref through getPersonaProps (for example page I used getPersonaProps() inside Facepile.Basic.Example.tsx), however it was never triggered.
I think it gets passed down to Persona.tsx - I saw it in this.props inside Persona's render function. But then it's not assigned to any explicit variable, and divProps doesn't cover it.

Does that make sense? Maybe a fix in Persona.tsx could remedy it and keep the code cleaner?